src/app/public/testing/phonefield/phone-field-fixture.ts

```diff
@@ -0,0 +1,147 @@
+import {
+  DebugElement
+} from '@angular/core';
+
+import {
+  ComponentFixture
+} from '@angular/core/testing';
+
+import {
+  By
+} from '@angular/platform-browser';
+
+import {
+  SkyAppTestUtility
+} from '@skyux-sdk/testing';
+
+import {
+  SkyCountryFieldFixture
+} from '@skyux/lookup/testing';
+
+/**
+ * Provides information for and interaction with a SKY UX phone field component.
+ * By using the fixture API, a test insulates itself against updates to the internals
+ * of a component, such as changing its DOM structure.
+ */
+export class SkyPhoneFieldFixture {
+  // this property is lazy loaded and should be accessed via the private countryFixture property
+  private _countryFixture: SkyCountryFieldFixture;
+  private _debugEl: DebugElement;
+
+  /**
+   * The value of the input field for the phone field.
+   */
+  public get inputText(): string {
+    return this.phoneFieldInput.value;
+  }
+
+  constructor(
+    private fixture: ComponentFixture<any>,
+    private skyTestId: string
+  ) {
+    this._debugEl = SkyAppTestUtility
+      .getDebugElementByTestId(fixture, skyTestId, 'sky-phone-field');
+
+    // The country selector needs extra time to initialize.
+    // Consumers shouldn't need to work around this so we do an extra detect here
+    fixture.detectChanges();
+  }
+
+  /**
+   * Sets the value of the input field for the phone field.
+   */
+  public async setInputText(inputText: string): Promise<any> {
+    const inputEl = this.phoneFieldInput;
+    inputEl.value = inputText;
+
+    SkyAppTestUtility.fireDomEvent(inputEl, 'input');
+    this.fixture.detectChanges();
+
+    SkyAppTestUtility.fireDomEvent(inputEl, 'change');
+
+    this.fixture.detectChanges();
+    return this.fixture.whenStable();
+  }
+
+  /**
+   * Opens the country selector, peforms a search, but makes no selection.
+   * @param searchText The name of the country to select.
+   */
+  public async searchCountry(searchText: string): Promise<NodeListOf<HTMLElement>> {
```

Comment:
Do we need this method? I feel like selecting the country is sufficient, unless you're meeting a different requirement?

Reply:
I figured people might care to simply open the search results and test against how they overlay other elements for whatever use cases they have. We exposed the same thing on the country-field, so it seemed logical to do the same here.

If you want me to get rid of it I can.

Reply:
I would argue that it's not something we should have exposed for country field either, because...

My biggest concern is that it returns HTML elements after the search which locks us down to a specific HTML format (and couldn't be changed without potentially breaking the consumer's unit tests). For example, if the consumer runs `searchCountry` and then executes some DOM query on the results, we wouldn't be able to change those DOM elements in the future because the consumer is now reliant on those queries for their test to pass.

If you really think there's a use-case for a consumer to check against search results, I would return an array of strings representing each country so that the DOM is still hidden from the end consumer.

(I would actually advise that we change--e.g. "fix"--the country field fixture to not return a node list, but just an array of country names.)

Reply:
The country updates should be reflected here now. I've also added a unit test specific to this use case.